src/Service/ActivityPubManager.php

```diff
@@ -741,16 +743,20 @@ private function handleMagazineFeaturedCollection(string $actorUrl, Magazine $ma
                         if (!$alreadyPinned) {
                             $existingEntry = $this->entryRepository->findOneBy(['apId' => $apId]);
                             if ($existingEntry) {
-                                $this->logger->debug('pinning existing entry: {title}', ['title' => $existingEntry->title]);
+                                $this->logger->debug('Pinning existing entry: {title}', ['title' => $existingEntry->title]);
                                 $this->entryManager->pin($existingEntry, null);
                             } else {
                                 $object = $item;
                                 if ($isString) {
-                                    $this->logger->debug('getting {url} because we dont have it', ['url' => $apId]);
-                                    $object = $this->apHttpClient->getActivityObject($apId);
+                                    if (!$this->settingsManager->isBannedInstance($apId)) {
+                                        $this->logger->debug('Getting {url} because we dont have it', ['url' => $apId]);
+                                        $object = $this->apHttpClient->getActivityObject($apId);
+                                        $this->logger->debug('Dispatching create message for entry: {e}', ['e' => json_encode($object)]);
+                                        $this->bus->dispatch(new CreateMessage($object, true));
```

Comment:
This has to be outside of the `if ($isString)` 

It can and should be behind its own `if (!$this->settingsManager->isBannedInstance($apId))`

Reply:
You mean this? https://github.com/MbinOrg/mbin/pull/1254/commits/042ab8b433f1abc170ae33d52fb1d29ead3db945

Reply:
exactly :+1:

Reply:
In that case, can't we just wrap it all behind a single `isBannedInstance` check?

Reply:
would work as well